public/components/trace_analytics/components/common/__tests__/helper_functions.test.tsx

```diff
@@ -94,9 +93,15 @@ describe('Helper functions', () => {
     expect(serviceMapGraph).toEqual(TEST_SERVICE_MAP_GRAPH);
   });
 
-  it('returns target resources by service name', () => {
-    const targetResources = getServiceMapTargetResources(TEST_SERVICE_MAP, 'order');
-    expect(targetResources).toEqual(['clear_order', 'update_order', 'get_order', 'pay_order']);
+  it('extracts correct target resources for a given service from its traceGroups', () => {
+    const traceGroups = TEST_SERVICE_MAP.order.traceGroups;
+
+    // Extract all target resources from the traceGroups
+    const targetResources = [].concat(...traceGroups.map((group) => group.targetResource));
+
+    // Verify the extracted resources match the expected list
+    const expectedResources = ['clear_order', 'update_order', 'get_order', 'pay_order'];
+    expect(targetResources).toEqual(expectedResources);
   });
```

Comment:
remove this test? it's not really testing anything

Reply:
Removed